### PR TITLE
[RFC][NI] Add autocomplete

### PR DESCRIPTION
**Pull request by montoias**
Add modal & and change to info-card

Add clickoutside mixin

### CODE REVIEW

File: app/components/uni-autocomplete-no-results.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-uniq/components/uni-autocomplete-no-results';
```
Comment (mmelo) [inline app/components/uni-autocomplete-no-results.js]: add newline please

File: app/components/uni-info-card.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-cli-uniq/components/uni-info-card';
```
Comment (mmelo) [inline app/components/uni-info-card.js]: add newline please

**Review by mmelo — CHANGES_REQUESTED** [review]
reviewed

File: app/components/uni-autocomplete-no-results.js (hunk 1 above)
Comment (AndreJoaquim) [inline app/components/uni-autocomplete-no-results.js]: newline

File: app/components/uni-info-card.js (hunk 2 above)
Comment (AndreJoaquim) [inline app/components/uni-info-card.js]: newline

**Comment by AndreJoaquim** [comment]
reviewd

**Review by mmelo — APPROVED** [review]

**Pull request merged**